Cargo.toml

```diff
@@ -32,6 +32,7 @@ global-context = ["std"]
 # if you are doing a no-std build, then this feature does nothing
 # and is not necessary.)
 global-context-less-secure = ["global-context"]
+silentpayments-example = ["rand", "std", "libc", "cc"]
```

Comment:
This should declare `required-features` instead.

Reply:
Done. Thanks for the review.